ckanext/xloader/tests/test_jobs.py

```diff
@@ -81,7 +81,7 @@ def test_xloader_data_into_datastore(self, cli, data):
         with mock.patch("ckanext.xloader.jobs.get_response", get_response):
             stdout = cli.invoke(ckan, ["jobs", "worker", "--burst"]).output
             assert "File hash: d44fa65eda3675e11710682fdb5f1648" in stdout
-            assert "Fields: [{'id': 'x', 'type': 'text'}, {'id': 'y', 'type': 'text'}]" in stdout
+            assert "Fields: [{'id': 'x', 'type': 'text', 'strip_extra_white': True}, {'id': 'y', 'type': 'text', 'strip_extra_white': True}]" in stdout
```

Comment:
We also need tests of the ability to disable this behaviour.

Reply:
@ThrawnCA okay added test coverage for `load_csv` and `load_table` with `'strip_extra_white': False`

Reply:
...welp those didn't work. Worked locally on 2.9 and 2.11, will see if I can replicate and fix em up

Reply:
@ThrawnCA I cannot seem to be able to replicate the failed tests in any of my environments.

Reply:
@JVickery-TBS  can you rebase this and try again to see if the 'tests' fix themselves.